### PR TITLE
Update dropkick.js

### PR DESCRIPTION
**Pull request by vreshch**
Some mobiles  gets confused on refresh call

### CODE REVIEW

**Comment by Robdel12** [comment]
Can you write a test proving the bug and that this fixes it?

File: dropkick.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ var
     var i;
 
     // Prevent DK on mobile
-    if ( window.isMobile && !opts.mobile ) {
+    if ( window.isMobile && opts &&!opts.mobile ) {
```
Comment (Robdel12) [inline dropkick.js]: `window.isMobile && opts && !opts.mobile`

**Comment by spoonben** [comment]
I've actually also seen this, I'll try writing the test to prove it.

**Comment by wwilsman** [comment]
Could you explain why it would be better? As I see it, besides that if statement, `opts` is only ever really passed onto the init method, which handles it appropriately. It'd be redundant to have both `this.opts` and `this.data.settings` (which also holds other data as well).

**Comment by spoonben** [comment]
@wwilsman, thought this was a better way to test https://github.com/Robdel12/DropKick/pull/266, also I don't think it is redundant since settings is extending opts https://github.com/Robdel12/DropKick/blob/master/dropkick.js#L259

**Pull request merged**